Makefile

```diff
@@ -1,28 +1,41 @@
-all: evm_precompiles evm_engines wasm_engines scout_engines notebook
-
-build_docker_images:
-	cd evm/geth && docker build . -t geth-bench
-	cd evm/parity && docker build . -t parity-bench
-	cd evm/evmone && docker build . -t evmone-bench
-	cd evm/cita-vm && docker build . -t cita-vm-bench
-	cd evm/evmone-evm384 && docker build . -t evmone-evm384-bench
+all: build_docker_images benchmark notebook
+
+build_evm_engines:
+	cd evm/geth && docker build . -t ewasm/geth-bench:1
```

Comment:
Do you want to create a PR which changes the makefile and scripts to use `ewasm/` ?

Reply:
is there a reason not to just include them here?  I don't see the scope of this PR expanding beyond what it currently is.

Reply:
This PR already does multiple things:
1) changes a lot of the scripts/makefiles
2) runs new benchmarks and regenerates the notebook

Reply:
ok I'll split it out.